rafs/src/metadata/direct_v6.rs

```diff
@@ -640,16 +640,15 @@ impl OndiskInodeWrapper {
         Ok(chunks)
     }
 
-    fn find_target_block(&self, name: &OsStr) -> Result<usize> {
+    fn find_target_block(&self, name: &OsStr) -> Result<Option<usize>> {
         let inode = self.disk_inode();
         if inode.size() == 0 {
-            return Err(enoent!());
```

Comment:
so when we get Err()?

Reply:
There are several `map_err()?` in the function body.

Reply:
And the caller `get_child_by_name()` while return `enoent()` if `find_target_block()` returns `Ok(None)`.